DESIGN.md

```diff
@@ -10,6 +10,10 @@ A few principles to guide the development and design of the game.
   robots at things by brute force (though it's OK if some brute force
   is required as well).
 
+- Don't provide complex built-in features; instead, provide simple,
+  fundamental, composable pieces out of which users can build the
+  complex features themselves.
```

Comment:
@byorgey could you please split this into sections?

For example:
> ### Composability
> Don't provide complex built-in features; instead, provide simple, fundamental, composable pieces out of which users can build the complex features themselves.

As a bonus, it will be easier to add examples and longer explanations into sections than to correctly indent them in nested bullet points. :slightly_smiling_face:

Reply:
Good idea, will do!